### PR TITLE
cal: add myles to maintainer list

### PR DESCRIPTION
**Pull request by MylesBorins**
Closes: #252

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ The calendar is maintained by:
 - [@maddhruv](https://github.com/maddhruv) - **Dhruv Jain**
 - [@mcollina](https://github.com/mcollina) - **Matteo Collina**
 - [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**
+- [@MylesBorins][https://github.com/MylesBorins] - **Myles Borins**
```
Comment (mhdawson) [inline README.md]: Second brackets need to be () instead of []

**Comment by MylesBorins** [comment]
was I actually added as an admin? Unable to add things to cal right now afaict

**Comment by dshaw** [comment]
TLC Chairs have access to add Myles. @mhdawson is your Chair.

Can you add @MylesBorins, @mhdawson?

**Comment by mhdawson** [comment]
Added Myles, he should get an invite and be able to modify the calendar now.

**Pull request merged**